crates/swc_ecma_preset_env/tests/fixtures/corejs3/usage-finished-proposal/options.json

```diff
@@ -4,7 +4,7 @@
       "../../../../lib",
       {
         "useBuiltIns": "usage",
-        "corejs": "3.21",
```

Comment:
Please add another test and revert this file

Reply:
I reverted!